### PR TITLE
Do not call cudnnGetVersion on import

### PR DESCRIPTION
**Pull request by kmaehashi**
This is harmless (so far) but it's better to avoid calling CUDA APIs on import.

### CODE REVIEW

**Review by leofang — APPROVED** [review]

**Review by takagi — APPROVED** [review]

**Comment by takagi** [comment]
pfnCI, test this please.

**Comment by takagi** [comment]
pfnCI, test this please.

File: cupy/cudnn.pyx [hunk 1]
```diff
@@ -1588,7 +1597,7 @@ cpdef _Algorithm _get_algorithm_bwd_data(
         return algo
     cdef list ret
     cdef bint skip
-    if _cudnn_version >= 7000:
+    if cudnn_version() == 7000:
```
Comment (takagi) [inline cupy/cudnn.pyx:1600]:
```suggestion
    if cudnn_version() >= 7000:
```

**Comment by takagi** [comment]
pfnCI, test this please.

**Review by takagi — APPROVED** [review]

**Comment by takagi** [comment]
Will be merged after code-freeze cleared.

File: cupy/cudnn.pyx (hunk 1 above)
Comment (kmaehashi) [reply cupy/cudnn.pyx]: Thanks, nice catch!

**Pull request merged**